archive/dug_6/dug_6_dasharo_configuration_utility_status_update.md

```diff
@@ -0,0 +1,257 @@
+class: center, middle, intro
+
+# Dasharo Configuration Utility Status Update
+
+### DUG #6 2024
+
+## Tymoteusz Burak
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+---
+
+# Agenda
+
+* Reminder: What is DCU?
+* What we envisioned
+* Current Status
+* The problem
+* Our solution
+* What does this mean to Dasharo users
+* Demo
+* Q&A
+
+???
+
+- Here is the Agenda
+
+---
+
+# Reminder: What is DCU?
+
+_The Dasharo Configuration Utility is a tool designed to configure Dasharo firmware binary images._
+
+#### Until now there was support for:
+- Customizing the boot logo
+- Setting unique UUIDs or Serial Numbers in SMBIOS tables
+
+???
+
+- Our dedicated tool that enabled customization of Dasharo binaries for users
+- Published around 7 months ago
+- Introduced at Dasharo User Group #4
+- Upon release allowed only for customization of the boot splash logo and
+  managing the Serial Numbers for devices
+
+---
+
+# Reminder: What is DCU?
+
+_The Dasharo Configuration Utility is a tool designed to configure Dasharo firmware binary images._
+
+.left-column50[
+.code-15px[
+```sh
+$ git clone https://github.com/Dasharo/dcu.git
+$ cd dcu
+$ ./dcuc logo coreboot.rom -l bootsplash.bmp
+```
+]
+]
+
+.right-column50[
+.center[<img src="/img/custom_boot.jpg" width="399px">]
```

Comment:
This can be a separate slide on the demo/showcase of bootsplash feature - command + result

Reply:
I wanted to highlight what are the new features of DCU and changing the boot splash logo has been one of its core features from the start but I can add this to the showcase as well

Reply:
Decided to stick to the original intent and the Example scenario covers setting of serial communication.